vertx-config/src/main/java/io/vertx/config/impl/spi/HttpConfigStoreFactory.java

```diff
@@ -44,9 +45,11 @@ public ConfigStore create(Vertx vertx, JsonObject configuration) {
     String host = configuration.getString("host");
     int port = configuration.getInteger("port", 80);
     String path = configuration.getString("path", "/");
+    MultiMap headers = MultiMap.caseInsensitiveMultiMap();
+    configuration.getJsonObject("headers", new JsonObject()).forEach(h -> headers.set(h.getKey(), h.getValue().toString()));
```

Comment:
in case there is a null value, can you filter the headers entries to skip those ?

Reply:
Sure, will do that.